css/build.css

```
@@ -0,0 +1,3 @@
+.inline-link {
```

You don't need to add a new class to the link. The widget has a container that provides sufficient context. It also means developers can have an element with a custom `inline-link` class without being affected.

```suggestion
[data-widget-package="com.fliplet.inline-link"] > a {
```

---

fixed